client/app/styles/less/sf-icons.less

```diff
@@ -523,6 +523,67 @@ a:hover .big-icon-info {
+}
+    .active {
+        .big-icon-unspike {
+            background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjhweCIgaGVpZ2h0PSIyNHB4IiB2aWV3Qm94PSIwIDAgMjggMjQiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSIgc3R5bGU9ImZpbGwtcnVsZTpldmVub2RkO2NsaXAtcnVsZTpldmVub2RkO3N0cm9rZS1saW5lam9pbjpyb3VuZDtzdHJva2UtbWl0ZXJsaW1pdDoxLjQxNDIxOyI+CiAgICA8ZyBpZD0iTGF5ZXIgMSI+CiAgICAgICAgPHBhdGggZD0iTTIxLDRMMTYuOCw0QzE2LjQsMi44IDE1LjMsMiAxNCwyQzEyLjcsMiAxMS42LDIuOCAxMS4yLDRMNyw0QzYuNCw0IDYsNC40IDYsNUw2LDZMMjIsNkwyMiw1QzIyLDQuNCAyMS42LDQgMjEsNCIgc3R5bGU9ImZpbGw6cmdiKDAsMTI3LDE3OSk7ZmlsbC1ydWxlOm5vbnplcm87Ii8+CiAgICAgICAgPGc+CiAgICAgICAgICAgIDxnIG9wYWNpdHk9IjAuMzk5OTk0Ij4KICAgICAgICAgICAgICAgIDxwYXRoIGQ9Ik05LDEzLjM1NEw5LDIwTDExLDIwTDExLDE1LjM1NEw5LDEzLjM1NFoiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgICAgIDxwYXRoIGQ9Ik04LjQ0LDdDOC4yNzgsNyA4LjE5Niw3LjE5NiA4LjMxMSw3LjMxMUwxMCw5TDExLDlMMTEsMTBMMTMsMTJMMTMsOUwxNSw5TDE1LDE0TDE3LDE2TDE3LDlMMTksOUwxOSwxOEwyMC42NDgsMTkuNjQ3QzIwLjc3OCwxOS43NzcgMjEsMTkuNjg1IDIxLDE5LjUwMUwyMSw3TDguNDQsN1oiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgIDxwYXRoIGQ9Ik0xNSwyMEwxMywyMEwxMywxNy40MDNMMTUsMTkuNDAzTDE1LDIwWk05LDIwTDksMTMuNDAzTDExLDE1LjQwM0wxMSwyMEw5LDIwWk0yMC43MTIsMjIuMzA1TDUuNjkzLDcuMjg3QzUuMzA3LDYuOSA0LjY4LDYuOSA0LjI5NCw3LjI4N0w0LjI4OCw3LjI5MkMzLjkwMiw3LjY3OSAzLjkwMiw4LjMwNSA0LjI4OCw4LjY5Mkw3LDExLjQwM0w3LDIwQzcsMjEuMSA3LjksMjIgOSwyMkwxNy41OTcsMjJMMTkuMzA3LDIzLjcxQzE5LjY5MywyNC4wOTcgMjAuMzIsMjQuMDk3IDIwLjcwNiwyMy43MUwyMC43MTIsMjMuNzA1QzIxLjA5OCwyMy4zMTggMjEuMDk4LDIyLjY5MiAyMC43MTIsMjIuMzA1IiBzdHlsZT0iZmlsbDpyZ2IoMCwxMjcsMTc5KTtmaWxsLXJ1bGU6bm9uemVybzsiLz4KICAgICAgICA8Zz4KICAgICAgICAgICAgPGcgb3BhY2l0eT0iMC4zOTk5OTQiPgogICAgICAgICAgICAgICAgPHBhdGggZD0iTTE1LDE0TDE1LDlMMTMsOUwxMywxMkwxNSwxNFoiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgICAgIDxnPgogICAgICAgICAgICA8ZyBvcGFjaXR5PSIwLjM5OTk5NCI+CiAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMTMsMTcuMzQ0TDEzLDIwTDE1LDIwTDE1LDE5LjM0NEwxMywxNy4zNDRaIiBzdHlsZT0iZmlsbDpyZ2IoMCwxMjcsMTc5KTtmaWxsLXJ1bGU6bm9uemVybzsiLz4KICAgICAgICAgICAgPC9nPgogICAgICAgIDwvZz4KICAgICAgICA8Zz4KICAgICAgICAgICAgPGcgb3BhY2l0eT0iMC4zOTk5OTQiPgogICAgICAgICAgICAgICAgPHBhdGggZD0iTTE5LDE4TDE5LDlMMTcsOUwxNywxNkwxOSwxOFoiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgICAgIDxnPgogICAgICAgICAgICA8ZyBvcGFjaXR5PSIwLjM5OTk5NCI+CiAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMTAuMDE2LDlMMTEsOS45ODRMMTEsOUwxMC4wMTYsOVoiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgPC9nPgo8L3N2Zz4K);
```

Comment:
Just curious - why are all these icons defined inline? Is this better than using a sprite?